shared/actions/push.js

```diff
@@ -132,6 +141,16 @@ function* pushNotificationSaga(notification: PushGen.NotificationPayload): Saga.
           logger.error('Push chat notification payload missing conversation ID')
           break
         }
+        // If a boxed message is attached to the notification, unbox.
+        if (payload.m) {
+          logger.info('Push notification: unboxing notification message')
+          yield Saga.call(RPCChatTypes.localUnboxMobilePushNotificationRpcPromise, {
+            convID: convID || '',
```

Comment:
these || '' aren't necessary anymore right?

Reply:
i left them in since i was getting some flow errors:

```
Error ------------------------------------------------------------------------------------------- actions/push.js:147:17

Cannot call `Saga.call` because string [1] is incompatible with undefined [2] in property `payload` of the first
argument of type argument `Fn`.

   actions/push.js:147:17
                        v-------------------------------------------------------------------
   147|           yield Saga.call(RPCChatTypes.localUnboxMobilePushNotificationRpcPromise, {
   148|             convID: convID,
   149|             membersType,
   150|             payload: payload.m,
   151|             shouldAck: false,
   152|           })
                  -^

References:
   constants/types/rpc-chat-gen.js:557:76
   557| export type LocalUnboxMobilePushNotificationRpcParam = $ReadOnly<{payload: String, convID: String, membersType: ConversationMembersType, pushIDs?: ?Array<String>, shouldAck: Boolean}>
                                                                                   ^^^^^^ [1]
   constants/types/push.js:10:7
    10|   m?: string,
              ^^^^^^ [2]



Found 3 errors
```

Reply:
this is because flow doesn't know if there's logic inbetween which could have mutated the state of the values you already checked. aka if you did 
```
const {convID, m} = payload
then passed m it'd work
```